### DISCUSSION TITLE
3.2.2 Bug Fix build publish to NPM?

### DISCUSSION THREAD

**Discussion by blitzve0** [weareseeed/react-square-web-payments-sdk#102] [General]
So this bug has been here for a long time, and now its finally fixed can someone let me know if this will be published to NPM anytime soon, if its not in plans i can fork and locally include the source code
But its very helpful if this can be published to NPM, Thank you

**Comment by easydoesit** [comment]
I'm looking for this update from NPM as well.

**Comment by taylor-shift** [comment]
Same here.
